test/modules/graphstorage.test.js

```diff
@@ -38,35 +38,60 @@ describe('GraphStorage module', () => {
             'host', 'port', 'max_path_length', 'database']);
         selectedDatabase.database = myDatabaseName;
 
-        systemDb = new Database();
-        systemDb.useBasicAuth('root', 'root');
-        await systemDb.createDatabase(
-            myDatabaseName,
-            [{ username: myUserName, passwd: myPassword, active: true }],
-        );
+        if (selectedDatabase.database_system === 'arango_db') {
```

Comment:
call graph storage `connect()` which needs to initialize the db according to the configuration